back/spec/acceptance/projects_spec.rb

```diff
@@ -607,6 +621,29 @@
         expect(json_response[:data].size).to eq 1
       end
 
+      example "Search for projects does not return projects with draft status" do
+        p1 = create(:project,
+              admin_publication_attributes: { publication_status: "published" },
+              title_multiloc: {
+                "en": "super-specific-title-string-1",
+                "fr-BE": "a title",
+                "nl-BE": "a title"
+              })
+
+        p2 = create(:project,
+          admin_publication_attributes: { publication_status: "draft" },
+          title_multiloc: {
+            "en": "super-specific-title-string-2",
+            "fr-BE": "a title",
+            "nl-BE": "a title"
+          })
+
+        do_request search: "super-specific-title-string"
+        json_response = json_parse(response_body)
+        expect(json_response[:data].size).to eq 1
+        expect(json_response[:data][0][:id]).to eq p1.id
```

Comment:
The comment was saying that we have helpers for that.
```suggestion
    expect(response_data.size).to eq 1
    expect(response_ids).to eq [p1.id]
```
if I'm not mistaken.

Reply:
~~Not sure I understand. Helpers for what?~~

Reply:
Oh, I see 🤦 You actually suggested the helpers to use!